sql_generators/glean_usage/templates/macros.sql

```diff
@@ -21,20 +20,14 @@ _core AS (
 _core_clients_first_seen AS (
   SELECT
     _fennec_id_lookup.client_id,
-    first_seen_date
+    MIN(first_seen_date) AS first_seen_date,
```

Comment:
this PRs allow the `_fennec_id_lookup` CTE to contain duplicates, and instead removes them at this stage, in order to get a more correct `first_seen_date` given the assumption that `migration_table` is allowed to produce a many to many relationship